test/__snapshots__/ecma.test.js.snap

```diff
@@ -106,7 +106,7 @@ exports[`ecma 6: errors 1`] = `Array []`;
 
 exports[`ecma 6: main.js 1`] = `
 "webpackJsonp([ 0 ], [ function(module, exports) {
-    class Mortgage {
+    module.exports = class {
```

Comment:
Why did this change ? It's the same as in #190 I just can't reason about it 😛

Reply:
@michael-ciniawsky oh yep :+1: